web/crux-ui/playwright.config.ts

```diff
@@ -8,7 +8,7 @@ const baseURL = process.env.E2E_BASE_URL || 'http://172.17.0.1:8000'
 const config: PlaywrightTestConfig = {
   globalSetup: path.join(__dirname, 'e2e', 'utils', 'global-setup.ts'),
   globalTeardown: path.join(__dirname, 'e2e', 'utils', 'global-teardown.ts'),
-  timeout: 120 * 1000, // 2 min
+  timeout: 2 * 60 * 1000, // 2 min
```

Comment:
we can adjust this back to 30 sec probably? may 45 if that's more reliable

Reply:
Yeah, but then we have to drop the worker count to two or one.

Reply:
whichever is faster overall

Reply:
The optimum is somewhere around 2-4 workers, depending on the weather, time, and constellations. Dropped to 2 workers rn, and was timing out on 45s too.

Reply:
2 min timeout with 3 workers seems pretty solid. Rebased, let's see it again.